### ISSUE TITLE
Node.js 16 actions are deprecated

### ISSUE THREAD

**Issue by rmclaughlin-nelnet** [rapid7/insightappsec-scan-github-actions#41]
We are seeing this warning when using this action. Can you please update your library?
> Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: rapid7/insightappsec-scan-github-actions@v1.3.3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.

**Comment by salbracco24** [comment]
Yes, please update it, we pay for this product, and it's an easy fix.
